src/Groups/libraries/atlasgroups.jl

```diff
@@ -0,0 +1,77 @@
+export
+    atlas_group,
+    number_atlas_groups
+
+###################################################################
+# Groups from the Atlas of Group Representations
+###################################################################
+
+"""
+    atlas_group([::Type{T}, ]name::String) where T <: Union{PermGroup, MatrixGroup}
+
+Return a group from the Atlas of Group Representations
+whose isomorphism type is given by `name` and have the type `T`.
+
+# Examples
+```jldoctest
+julia> atlas_group("A5")  # alternating group A5
+Group([ (1,2)(3,4), (1,3,5) ])
+
+julia> atlas_group(MatrixGroup, "A5")
+Matrix group of degree 4 over Galois field with characteristic 2
+
+julia> atlas_group("M11")  # Mathieu group M11
+Group([ (2,10)(4,11)(5,7)(8,9), (1,4,3,8)(2,5,6,9) ])
+
+```
+"""
+function atlas_group(name::String)
+  G = GAP.Globals.AtlasGroup(GAP.GapObj(name))
+  G === GAP.Globals.fail && error("the group atlas does not provide a representation for $name")
+  T = _get_type(G)
+  return T(G)
+end
+
+function atlas_group(::Type{T}, name::String) where T <: Union{PermGroup, MatrixGroup}
+  if T === PermGroup
+    G = GAP.Globals.AtlasGroup(GAP.GapObj(name), GAP.Globals.IsPermGroup, true)
+  else
```

Comment:
Maybe check whether a MatrixGroup is requested and otherwise produce an error?

Also, perhaps you could use `_gap_filter` here?

Reply:
This is anyhow a very preliminary piece of code. Eventually we want to specify which kind of matrix group we want to get.